src/systems/rendering.cpp

```diff
@@ -120,6 +120,13 @@ void drawMesh(MeshRenderComponent& mrc, ShaderRegistry::ScopedShader& shader)
 
 void MeshRenderSystem::update(float delta)
 {
+    // Update banking angle for all physics entities.
+    for (auto [entity, mrc, transform, physics] : sp::ecs::Query<MeshRenderComponent, sp::Transform, sp::Physics>())
+    {
+        float target_bank_angle = 0.0f;
+        target_bank_angle = std::clamp(physics.getAngularVelocity(), -4.0f, 4.0f);
```

Comment:
Won't this cause things like asteroids (that always rotate) to bank as well?

Reply:
I thought the Spin component set rotation directly each tick independent of physics, instead of setting angular velocity:

https://github.com/daid/EmptyEpsilon/blob/9405eaf272e1f7e92be6808e1dcc63e8987f88cb/src/systems/basicmovement.cpp#L16-L18

But I can narrow this PR's component selection further to the ManeuveringThrusters component